PlayFabSDK/source/Qos/QosRegionResult.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace PlayFab.QoS
+{
+    using MultiplayerModels;
+
+    public class QosRegionResult
+    {
+        public AzureRegion Region { get; set; }
```

Comment:
A { get; set; } property is syntactically identical to a field, but about 1000x more expensive than fields.
I suggest only using properties when there is a meaningful reason to do so (inheritance for example).

Reply:
Would you point me some reference about performance benchmark using property vs field? From my knowledge, an auto property is the same as a field in performance. Properties are usually exposed as public and field is more used privately for packaging and extension purpose.